src/EVM/Assembler.hs

```diff
@@ -0,0 +1,104 @@
+{-# LANGUAGE DataKinds #-}
```

Comment:
Not a big deal, but it would be nice to have a [module description](https://haskell-haddock.readthedocs.io/en/latest/markup.html#the-module-description) here. Just the module and name fileds is fine.

Reply:
Thanks! Yeah, that's a great idea. I just added it in a commit, will push and resolve ;)